### ISSUE TITLE
Agricraft 2.0.0-0.9.0-a16 requiring Agricraft

### ISSUE THREAD

**Issue by Gkoliver** [AgriCraft/AgriCraft#954]
The best I can do here is [to give you a link to the error message](https://i.redd.it/ul14z6xnmnyy.png)

**Comment by RlonRyan** [comment]
Well that's new...

**Comment by RlonRyan** [comment]
Does this happen every time you launch the game?

**Comment by Dictato-Volvox** [comment]
TL;DR - Check if you are missing infinitylib 1.10.2-0.8.0 or above

I'm having this same issue when I try with many other mods. It occurs every time I launch the game. Just to be sure, I have tried with 2.0.0-0.8.0 and I receive the same issue when a lot of other mods are applied. If you want, I can provide that full list of mods, and maybe if Gkoliver can (I dunno, I'm assuming you had other mods as well), it would be easier to narrow down the issue.

Here's the process that I used to solve this:

I tried 2.0.0-0.8.0 without any other mods and it gave me the error:

"Agricraft is missing mods it depends on.
Include the following mods or remove AgriCraft.

infinitylib 1.10.2-0.8.0 or above

See 'fml-client-latest.log' for technical information. "

Ok. That's a lot better than the original error message. So, I found a copy of infinity lib for 1.10.2 and minecraft booted up beautifully. But, that's the solution for 2.0.0-0.8.0. No mods, but it still booted up nonetheless.

So I then tried 2.0.0-0.9.0 with no mods, and infinitylib, minecraft booted up beautifully, with no errors. The only issue I had was the bad pun that a beta version was "betta". lol. So, at this point, both versions work without mods. I then tried adding my mods back in, and minecraft boots beautifully. The solution? adding infinitylib. Remaining problems? The Puns.

**Comment by RlonRyan** [comment]
How odd... I guess I'll have to check up on the forge documentation for required mods...

**Comment by RlonRyan** [comment]
Weird... The required-after syntax appears to be correct. I have no idea why this would be happening then.

The best I can say is that if the message shows up, try updating InfinityLib to match AgriCraft...
